src/main/scala/com/typesafe/sbt/web/package.scala

```diff
@@ -1,13 +1,15 @@
 package com.typesafe.sbt
 
+import PluginCompat.FileRef
+
 import java.io.File
 
 package object web {
 
   /**
    * Describes a string path relative to a base directory.
    */
-  type PathMapping = (File, String)
+  type PathMapping = (FileRef, String)
```

Comment:
This is necessary because `Keys.mappings` changed types in sbt 2. The type change creates significant type collisions in keys, mapping logic, and deduplicators.

Almost all of the type changes and `FileConverter` additions in `SbtWeb.scala` stem from this change.